public class MultiCatchExample {
    public static void main(String[] args) {
        try {
            int a = 10, b = 5;
            int result = a / b;  

            int[] arr = new int[5];
            arr[10] = 100; 

            String str = null;
            str.length(); 
        } catch (ArithmeticException | ArrayIndexOutOfBoundsException | NullPointerException e) {
            System.out.println("Exception caught: " + e);
        }
    }
}
